Tests/Build/DescribeTests.swift

```diff
@@ -25,19 +25,39 @@ final class DescribeTests: XCTestCase {
                 var clang: String { fatalError() }
             }
 
-            try POSIX.mkdtemp("spm-tests") { prefix in
-                defer { _ = try? Utility.removeFileTree(prefix) }
-                let _ = try describe(Path.join(prefix, "foo"), .debug, [], [], [], Xcc: [], Xld: [], Xswiftc: [], toolchain: InvalidToolchain())
+            let tempDir = try TemporaryDirectory(removeTreeOnDeinit: true)
+            _ = try describe(tempDir.path.appending("foo").asString, .debug, [], [], [], Xcc: [], Xld: [], Xswiftc: [], toolchain: InvalidToolchain())
                 XCTFail("This call should throw")
-            }
         } catch Build.Error.noModules {
             XCTAssert(true, "This error should be thrown")
         } catch {
             XCTFail("No other error should be thrown")
         }
     }
 
+    func testDescribingCModuleThrows() {
+        do {
+            struct InvalidToolchain: Toolchain {
+                var platformArgsClang: [String] { fatalError() }
+                var platformArgsSwiftc: [String] { fatalError() }
+                var sysroot: String?  { fatalError() }
+                var SWIFT_EXEC: String { return "" }
+                var clang: String { fatalError() }
+            }
+
+            let tempDir = try TemporaryDirectory(removeTreeOnDeinit: true)
+            _ = try describe(tempDir.path.appending("foo").asString, .debug, [CModule(name: "MyCModule", path: "")], [], [], Xcc: [], Xld: [], Xswiftc: [], toolchain: InvalidToolchain())
+                XCTFail("This call should throw")
+        } catch Build.Error.cModule (let name) {
+            XCTAssert(true, "This error should be thrown")
```

Comment:
Just copied and pasted from the test above; probably not needed indeed, but was included for clarity.